TemporalNeuralNetworks/Network/TemporalNeuralNetworks.cpp

```diff
@@ -65,6 +65,7 @@ void printStdpConfig(STDPConfigurator& stdpConfig)
 
 int main()
 {
+    try {
```

Comment:
This is not pretty but I had some issues at some point and since you dont have this anywhere to print your exceptions.